### DISCUSSION TITLE
How to disable public IP address associate for okd cluster

### DISCUSSION THREAD

**Discussion by veera418** [okd-project/okd#1624] [General]
I am trying to create okd on aws, I want to add disable public IP address in install-config yaml file.
So is there any way to disable public IP while creating okd cluster?

Version: 4.13
Cloud: AWS
Type: IPS

**Comment by JaimeMagiera** [comment]
In essence, this would be a private cluster. 

https://docs.okd.io/4.13/installing/installing_aws/installing-aws-private.html
